packages/tables/src/views/OverflowButton.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import TableStyles from '@zendeskgarden/css-tables';
+import { retrieveTheme } from '@zendeskgarden/react-theming';
+import { composeEventHandlers } from '@zendeskgarden/react-selection';
+
+const COMPONENT_ID = 'tables.overflow_button';
+
+/**
+ * Accepts all `<button>` props
+ */
+const StyledOverflowButton = styled.button.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  type: 'button',
+  className: props =>
+    classNames(TableStyles['c-table__row__cell__overflow'], {
+      [TableStyles['is-hovered']]: props.hovered,
+      [TableStyles['is-active']]: props.active,
+      [TableStyles['is-focused']]: props.focused
+    })
+})`
+  /**
+   * Must provide a width that is equivalent to the :before pseudo
+   * element to ensure correct Popper.js positioning
+   **/
+  min-width: 28px;
```

Comment:
Is this still necessary after reset removal of the button padding?

Reply:
Yes, this is necessary regardless of the type of element.

Since the only `:before` pseudo element is what is getting the width/height of the icon, Popper isn't able to understand this and applies relative to the edge, which throws of the horizontal alignment.

Rather than require all implementations this allows it to "just work" with the Menu component. Since this is implementation specific I feel comfortable leaving it on the React side for now.

Reply:
Resolves this on the css-tables side. Bumped the dependency to pull in the fix.